src/sections/fossil-fuel/UniversityInvestments.tsx

```diff
@@ -57,52 +58,55 @@ const UniversityInvestments: FC<UniversityInvestmentsProps> = ({
-      />
-      <div className="sm:px-12">
-        <div className="flex flex-col md:flex-row">
-          <PrimaryButton
-            text="Financed Emissions"
-            onClick={() => setFlag('financedEmissions')}
-            variant={
-              flag === 'financedEmissions' ? 'clementine-toggled' : 'clementine'
-            }
-            className="z-1 relative mb-5 w-full sm:mb-0 sm:w-fit"
-          >
-            {
-              <ToolTip
-                title="Financed Emission"
-                details={TOOLTIP_DEFINITIONS.FINANCED_EMISSIONS}
-                className="ml-1"
-              />
-            }
-          </PrimaryButton>
-          <PrimaryButton
-            text="Net Asset Value"
-            onClick={() => setFlag('netAssetValue')}
-            variant={
-              flag === 'netAssetValue' ? 'clementine-toggled' : 'clementine'
-            }
-            className="mb-5 w-full sm:mb-0 sm:w-fit"
-          >
-            {
-              <ToolTip
-                title="Net Asset Value"
-                details={TOOLTIP_DEFINITIONS.NET_ASSET_VAL}
-                className="ml-1"
-              />
-            }
-          </PrimaryButton>
-        </div>
+      <div className="mb-12 flex flex-wrap justify-center sm:mb-0">
+        <UnderlinedTitle
+          title="UI System Investments&nbsp;"
+          size="large"
+          color="white"
+        />
+        <UnderlinedTitle title="FY 2022" size="large" color="clementine" />
+      </div>
+      <div className="md:px-12">
         <div className="flex justify-center">
-          <div className="w-3/4">
-            <EmissionTreeMap flag={flag} />
+          <div className="w-full md:px-20 xl:px-40 2xl:px-64">
+            <div className="flex flex-col items-center gap-4 md:flex-row md:justify-start">
+              <Toggle
+                text="Financed Emissions"
+                onClick={() => setFlag('financedEmissions')}
+                toggled={flag === 'financedEmissions' ? true : false}
+                variant="cobalt"
+              >
+                {
+                  <ToolTip
+                    title={'Financed Emission'}
+                    details={TOOLTIP_DEFINITIONS.FINANCED_EMISSIONS}
+                  />
+                }
+              </Toggle>
+              <Toggle
+                text="Net Asset Value"
+                onClick={() => setFlag('netAssetValue')}
+                toggled={flag === 'netAssetValue' ? true : false}
+                variant="cobalt"
+              >
+                {
+                  <ToolTip
+                    title={'Net Asset Value'}
+                    details={TOOLTIP_DEFINITIONS.NET_ASSET_VAL}
+                  />
+                }
+              </Toggle>
+            </div>
+            <div className="flex justify-center">
+              <div className="w-full">
+                <EmissionTreeMap flag={flag} />
+              </div>
+            </div>
```

Comment:
```suggestion
            <div className="w-full">
              <EmissionTreeMap flag={flag} />
            </div>
```
Seems like some `div`s in this file can be merged together, can we check to make sure we are keeping `div` layers clean?